File: go/1134.go
package main

import "fmt"

func main() {
  var c int
  comb := map[int]int{
    1: 0,
    2: 0,
    3: 0,
  }

  for {
    fmt.Scanf("%d", &c)

    if c < 0 && c > 4 {
      continue
    }

    if c == 4 {
      break
    }

    comb[c] += 1
  }

  fmt.Printf("MUITO OBRIGADO\nAlcool: %d\nGasolina: %d\nDiesel: %d\n", comb[1], comb[2], comb[3])
}

File: lua/1134.lua
local comb = {0, 0, 0}

while true do
    local c = io.read("*n")

    if c >= 1 and c <= 4 then
      if c == 4 then
        break
      end
      comb[c] = comb[c] + 1
    end
end

print("MUITO OBRIGADO")
print(string.format("Alcool: %d", comb[1]))
print(string.format("Gasolina: %d", comb[2]))
print(string.format("Diesel: %d", comb[3]))
